runtime/westend/src/lib.rs

```diff
@@ -61,12 +62,7 @@ use session::historical as session_historical;
 
 #[cfg(feature = "std")]
 pub use staking::StakerStatus;
-#[cfg(any(feature = "std", test))]
-pub use sp_runtime::BuildStorage;
-pub use timestamp::Call as TimestampCall;
-pub use balances::Call as BalancesCall;
```

Comment:
Why do you remove them?

Reply:
> Also did a bit of cleanup in all runtime files, see here: https://github.com/paritytech/substrate/pull/6132#issuecomment-634509086

Reply:
Yeah that is for example used by Cumulus :D